apps/test/unit/code-studio/components/progress/teacherPanel/SelectedStudentInfoTest.jsx

```diff
@@ -72,27 +74,63 @@ describe('SelectedStudentInfo', () => {
     const levelWithProgress = {
       ...LEVEL_WITH_PROGRESS,
       paired: true,
+      isDriver: true,
       status: LevelStatus.perfect,
-      navigator: 'Student 2'
+      navigators: ['Student 2', 'Student 3']
     };
     const wrapper = setUp({levelWithProgress});
 
     expect(wrapper.contains('Worked With:')).to.equal(true);
     expect(wrapper.contains('Partner: Student 2')).to.equal(true);
+    expect(wrapper.contains('Logged in:')).to.equal(false);
+    expect(wrapper.contains('Last Updated:')).to.equal(true);
+  });
+
+  it('displays time and unknown navigator if paired as driver on level', () => {
+    const levelWithProgress = {
+      ...LEVEL_WITH_PROGRESS,
+      paired: true,
+      isDriver: true,
+      status: LevelStatus.perfect,
+      navigators: [null, 'Student 3']
+    };
+    const wrapper = setUp({levelWithProgress});
+
+    expect(wrapper.contains('Worked With:')).to.equal(true);
+    expect(wrapper.contains('Partner: n/a')).to.equal(true);
+    expect(wrapper.contains('Logged in:')).to.equal(false);
     expect(wrapper.contains('Last Updated:')).to.equal(true);
   });
 
   it('displays time and who they worked with as driver if paired as navigator on level', () => {
     const levelWithProgress = {
       ...LEVEL_WITH_PROGRESS,
       paired: true,
+      isNavigator: true,
       status: LevelStatus.perfect,
       driver: 'Student 2'
     };
     const wrapper = setUp({levelWithProgress});
 
     expect(wrapper.contains('Worked With:')).to.equal(true);
     expect(wrapper.contains('Logged in: Student 2')).to.equal(true);
+    expect(wrapper.contains('Partner:')).to.equal(false);
+    expect(wrapper.contains('Last Updated:')).to.equal(true);
+  });
+
+  it('displays time and unknown driver if paired as navigator on level', () => {
+    const levelWithProgress = {
+      ...LEVEL_WITH_PROGRESS,
+      paired: true,
+      isNavigator: true,
+      status: LevelStatus.perfect,
+      driver: null
+    };
+    const wrapper = setUp({levelWithProgress});
+
+    expect(wrapper.contains('Worked With:')).to.equal(true);
```

Comment:
Nit: these could be `.to.be.true` and `.to.be.false` instead of using `.equal`

Reply:
Yeah, agreed.  I was trying to match the style of the code that was already there.  Do you think I should go ahead and clean up the whole file?

Reply:
Up to you! I don't have a strong opinion